app/controllers/session.rb

```diff
@@ -0,0 +1,20 @@
+get '/session/new' do
+  erb :'/session/login'
+end
+
+
+post '/session' do
+  user = User.authenticate(params[:user])
```

Comment:
`params[:user]` should be `nil` here since the login form is not nesting params. (It is the `name` attributes in the login form that determine what each param will be named in the params hash.) Put a binding.pry here and inspect the params.

Reply:
FYI the lines in the login form that determine what these param keys will be are [here](https://github.com/nyc-red-pandas-2017/suggestion-box-crud/blob/1c4d8c6ba2f1c4989d9cd1c180371592be7078d9/app/views/session/login.erb#L47-L48). Specifically, it is the `name` attribute for each input that determines the key name in the params hash. (For example: If you change the name attribute on one of the input elements to "bananas", it will be params[:bananas] in the params hash.)